src/main/java/ca/intelliware/ihtsdo/mlds/service/UserMembershipAccessor.java

```diff
@@ -19,19 +19,20 @@
 @Service
 public class UserMembershipAccessor {
 
-	@Autowired
-	protected CurrentSecurityContext currentSecurityContext;
+	@Autowired protected CurrentSecurityContext currentSecurityContext;
 
-	@Autowired
-	AffiliateRepository affiliateRepository;
+	private final AffiliateRepository affiliateRepository;
 
-	@Autowired
-	MemberRepository memberRepository;
+    private final MemberRepository memberRepository;
 
-	@Autowired
-	AffiliateMembershipCalculator affiliateMembershipCalculator;
+	@Autowired AffiliateMembershipCalculator affiliateMembershipCalculator;
 
-	public Member getMemberAssociatedWithUser() {
+    public UserMembershipAccessor(AffiliateRepository affiliateRepository, MemberRepository memberRepository) {
+        this.affiliateRepository = affiliateRepository;
+        this.memberRepository = memberRepository;
+    }
```

Comment:
@jaykamal this looks like a mix of field and constructor injection. Can one style be used?

Reply:
@dmcgihtsdo @jimcornmell We have updated this constructor injection as per the recommendation.